dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/WorkspaceItemRestRepository.java

```diff
@@ -326,12 +330,19 @@ private void evaluatePatch(Context context, HttpServletRequest request, Workspac
                             " Therefore it cannot be used by the Configurable Submission as the <processing-class>!");
                     }
 
+                } catch (UnprocessableEntityException e) {
+                    log.error(e.getMessage(), e);
```

Comment:
Please log a more useful message here than e.getMessage(), e.g. the message in the line below